_config/security.yml

```diff
@@ -1,3 +1,27 @@
+---
+Name: cwppasswordstrength
+---
+# In the case someone uses `new PasswordValidator` instead of Injector
+# test names will not be set, as it is not configurable.
+SilverStripe\Security\PasswordValidator:
+  min_length: 10
+  min_test_score: 3
+  historic_count: 6
+# Set strength tests and requirements in line with NZISM
+# Injector is used by default for Member password validation
+SilverStripe\Core\Injector\Injector:
+  SilverStripe\Security\PasswordValidator:
+    class: SilverStripe\Security\PasswordValidator
```

Comment:
Is this needed? Without looking at anything it might be overriding it seems pointless

Reply:
Yeah I didn't look either, but memory serves that there's some funnyness around a missing class spec when other values are supplied (such as `properties`).
In all honesty I was rushing to get this through on change freeze day, and so a little redundancy wasn't the worst option.

Reply:
Ok, well would you mind removing it?